### PR TITLE
Added eslint rule New cap

### PR DESCRIPTION
**Pull request by LennyBoyatzis**
Progress commit

As per Review .eslintrc #1797

### CODE REVIEW

File: admin/client/components/Checkbox.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import blacklist from 'blacklist';
 import classnames from 'classnames';
-import Color from 'color';
+import colorConverter from 'color';
```
Comment (JedWatson) [inline admin/client/components/Checkbox.js]: couldn't this be `color`?

Comment (LennyBoyatzis) [inline admin/client/components/Checkbox.js]: There is already a variable called color which stores information about font color. It was either change this variable or the name of the color package. See code snippet

`		let checkedColor = colorConverter('#3999fc');

		let background = (checked && !readonly) ? checkedColor.hexString() : 'white';
		let borderColor = (checked && !readonly) ? 'rgba(0,0,0,0.15) rgba(0,0,0,0.1) rgba(0,0,0,0.05)' : 'rgba(0,0,0,0.3) rgba(0,0,0,0.2) rgba(0,0,0,0.15)';
		let boxShadow = (checked && !readonly) ? '0 1px 0 rgba(255,255,255,0.33)' : 'inset 0 1px 0 rgba(0,0,0,0.06)';
		let color = (checked && !readonly) ? 'white' : '#bbb';
		let textShadow = (checked && !readonly) ? '0 1px 0 rgba(0,0,0,0.2)' : null;
`

**Comment by mxstbr** [comment]
What's the blocker here?

**Comment by JedWatson** [comment]
Discussed w/ @LennyBoyatzis in person, we're going to bail on adding this eslint rule. I've come to think it's as much misdirection as clarification.

**Pull request closed without merging**